src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -538,7 +539,7 @@ function IOURequestStepConfirmation({
                     <HeaderWithBackButton
                         title={headerTitle}
                         onBackButtonPress={navigateBack}
-                        shouldShowThreeDotsButton={requestType === CONST.IOU.REQUEST_TYPE.MANUAL && (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.TRACK)}
```

Comment:
I tried replace the tracked expense receipt and BE will revert it back to the original receipt, so I hide the 3-dot button.